src/sensor/ping.cpp

```diff
@@ -30,6 +30,27 @@ Ping::Ping() : Sensor() {
 
     connect(&_detector, &ProtocolDetector::_detected, this, &Ping::connectLink);
     _detector.start();
+
+    connect(this, &Ping::autoDetectUpdate, this, [this](bool autodetect){
+        if(!autodetect) {
```

Comment:
if(connected) return;

Reply:
The idea behind this is to enable the auto detect function again, return if it's connected may results in problem where the serial device or udp was manually connected without a device to answer.

Reply:
There will be problems if we enter here and we are already talking to a device?